src/Sylius/Bundle/AttributeBundle/Form/EventSubscriber/BuildAttributeValueFormSubscriber.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Form\EventSubscriber;
+
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Sylius\Component\Registry\ServiceRegistryInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\FormFactoryInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class BuildAttributeValueFormSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var FormFactoryInterface
+     */
+    protected $factory;
+
+    /**
+     * @var string
+     */
+    protected $subjectName;
+
+    /**
+     * @var EntityRepository
+     */
+    protected $attributesRepository;
+
+    /**
+     * @param FormFactoryInterface $factory
+     * @param string $subjectName
+     * @param EntityRepository $attributesRepository
+     */
+    public function __construct(FormFactoryInterface $factory, $subjectName, EntityRepository $attributesRepository)
```

Comment:
I see that we only use ``find`` method, so we should typehint ``RepositoryInterface`` here.